security/file_integrity_operator/file-integrity-operator-configuring.adoc

```diff
@@ -0,0 +1,107 @@
+[id="configuring-file-integrity-operator"]
+= Configuring the Custom File Integrity Operator
+include::modules/common-attributes.adoc[]
+:context: file-integrity-operator
+toc::[]
+
+
+== Understanding the FileIntegrity object
+Let’s explore the . As usual with any Kubernetes CRs, you can call:
+$ oc explain fileintegrity
+And then look at the individual attributes:
+$ oc explain fileintegrity.spec
+$ oc explain fileintegrity.spec.config
+
+There are several attributes worth calling out in the spec and spec.config:
+spec.nodeSelector: A map of key-values pairs that must match with node’s labels in order for the AIDE pods to be schedulable on that node. The typical use is to set only a single key-value pair where node-role.kubernetes.io/worker: "" schedules aide on all worker nodes, "node.openshift.io/os_id": "rhcos" schedules on all RHCOS nodes etc.
+spec.debug: A boolean attribute, if set to true, the demon running in the AIDE deamonset’s pods would output extra information
+spec.tolerations: Specify tolerations to schedule on nodes with custom taints. When not specified, a default toleration allowing running on master nodes is applied.
+
+The .spec object further contains a config object:
+spec.config.gracePeriod: The number of seconds to pause in between AIDE integrity checks. Frequent AIDE checks on a node may be resource intensive, so it can be useful to specify a longer interval. Defaults to 3600 (one hour).
+spec.config.name, spec.config.namespace, spec.config.key: These three attributes allow you to set a custom AIDE configuration. When name or namespace are unset, the File Integrity Operator generates a configuration suitable for RHCOS systems. The name and namespace attributes point to the configmap, the key points to a key inside that configmap. The key attribute can be used to specify a custom key that contains the actual config and defaults to aide.conf
```

Comment:
the above references to the yaml keys (e.g. .spec.nodeSelector) would benefit from extra formatting (e.g. bold text)